src/CommonDBTM.php

```diff
@@ -485,22 +478,14 @@ public function showForm($ID, array $options = [])
         return true;
     }
 
-
-    public function getSNMPCredential()
-    {
-        if ($this->isField('snmpcredentials_id') && $this->fields['snmpcredentials_id']) {
-            $snmp_credential = new SNMPCredential();
-            $snmp_credential->getFromDB($this->fields['snmpcredentials_id']);
-            return $snmp_credential;
-        }
-        return false;
-    }
-
-
     /**
      * Retrieve locked field for the current item
      *
      * @return array
+     * @used-by templates/components/form/itemvirtualmachine.html.twig
```

Comment:
is this kind of documentation comments maintainable?

Reply:
It isnt necessarily meant to be all-encompassing, but having at least one used-by tag fixes issues with the IDE not seeing it used anywhere. At least for me, PHPStorm doesn't know when it is called from a Twig template.

Reply:
hum... OK.